Zend/zend_builtin_functions.stub.php

```diff
@@ -41,6 +41,8 @@ function strncasecmp(string $string1, string $string2, int $length): int {}
 
 function error_reporting(?int $error_level = null): int {}
 
+function exit_status(): int {}
```

Comment:
```suggestion
function exit_status(): ?int {}
```

Would make more sense to indicate we are in shutdown process or not.